visualization/settings/config.py

```diff
@@ -0,0 +1,26 @@
+from dataclasses import dataclass, field
+from typing import Dict, Optional, Tuple
+
+import hydra
+
+from nemo_skills.code_execution.utils import CODE_OUTPUT_SEPARATORS, CODE_SEPARATORS
+from nemo_skills.inference.generate_solutions import GenerateSolutionsConfig
+
+
+@dataclass
+class BaseVisualizationConfig:
+    prediction_jsonl_files: Dict[str, str] = field(default_factory=dict)
+
+    code_separators: Tuple[str, str] = CODE_SEPARATORS
+    code_output_separators: Tuple[str, str] = CODE_OUTPUT_SEPARATORS
+    dataset_path: Optional[str] = "datasets/{}/{}.jsonl"
```

Comment:
the default is confusing here, what do the brackets mean? Why don't you take dataset and split_name is parameters like in other files?